gapis/gfxapi/gfxapi.proto

```diff
@@ -50,6 +50,9 @@ enum ResourceType {
 	ShaderResource = 5;
 	// ProgramResource represents the Program resource type
 	ProgramResource = 6;
+	// UninitializedTextureResource represents texture resource type
+	// which has been generated but does not have dimensions yet.
+	UninitializedTextureResource = 7;
```

Comment:
Dimensions could change. Perhaps not for GLES, but it seems feasible for other APIs.
It seems wrong to special case this. What's wrong with reporting (0,0) or (-1,-1) as the dimensions?